addons/website_quote/controllers/main.py

```diff
@@ -113,61 +107,51 @@ def post(self, order_id, token, **post):
         return werkzeug.utils.redirect("/quote/%s/%s?message=1" % (order_id, token))
 
     def __message_post(self, message, order_id, message_type='comment', subtype=False, attachments=[]):
```

Comment:
can we pass order instead of order_id, so no need to browse order each time, as we already browse order in post, decline, accept methods.